.github/workflows/test_build.yml

```diff
@@ -160,7 +162,9 @@ jobs:
           make -C ../.. all
           export PYVERSION=`python3 -c "from sys import version_info; \
             print(f'{version_info.major}.{version_info.minor}')"`
-          export CPPFLAGS="-I`brew --prefix`/include -I`brew --prefix libomp`/include"
+          export CPPFLAGS="-I`brew --prefix`/include \
+            -I`brew --prefix libomp`/include         \
+            -I`brew --prefix flint`/include"
```

Comment:
Why is this necessary?

Reply:
I forget.  I'll try removing it and see what happens.